### PR TITLE
chore: update refs and build for tradeshift fork

### PR DESCRIPTION
**Pull request by wejendorp**

### CODE REVIEW

File: .github/workflows/pr-sonar.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Sonarqube scanner
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (sorenmat) [inline .github/workflows/pr-sonar.yml:10]:
```suggestion
    runs-on: self-hosted
```

**Review by sorenmat — APPROVED** [review]
other then that.. it looks good

File: .github/workflows/pr-sonar.yml (hunk 1 above)
Comment (sorenmat) [reply .github/workflows/pr-sonar.yml:10]: ahhh might be ok, since it's public

Comment (wejendorp) [reply .github/workflows/pr-sonar.yml:10]: afair self-hosted runners are not avail (yet?) for our public repos though.

**Pull request merged**